src/jdk.jdwp.agent/share/native/libjdwp/eventHandler.c

```diff
@@ -1509,11 +1504,6 @@ eventHandler_initialize(jbyte sessionID)
     if (error != JVMTI_ERROR_NONE) {
         EXIT_ERROR(error,"Can't enable thread end events");
     }
-    error = threadControl_setEventMode(JVMTI_ENABLE,
-                                       EI_CLASS_PREPARE, NULL);
-    if (error != JVMTI_ERROR_NONE) {
-        EXIT_ERROR(error,"Can't enable class prepare events");
-    }
```

Comment:
We no longer permanently enable CLASS_PREPARE events here. It's now done in classTrack.c, and only impacts the JVMTIEnv that us used for class tracking.